waywiser_hwbringup/package.xml

```diff
@@ -8,16 +8,26 @@
 
   <maintainer email="rickard.hall@ri.se">Rickard Häll</maintainer>
   <maintainer email="ramana.reddy.avula@ri.se">Ramana Avula</maintainer>
-  
+
   <license>TODO: License declaration</license>
 
   <url type="website">https://github.com/RISE-Dependable-Transport-Systems/WayWise</url>
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
+  <build_depend>rclcpp</build_depend>
+  <build_depend>sensor_msgs</build_depend>
+  <build_depend>tf2_ros</build_depend>
+  <build_depend>tf2_sensor_msgs</build_depend>
+  <build_depend>geometry_msgs</build_depend>
+
   <depend>joint_state_publisher</depend>
   <depend>robot_state_publisher</depend>
   <depend>rplidar_ros</depend>
+  <depend>librealsense2-utils</depend>
+  <depend>librealsense2-dev</depend>
+  <depend>realsense2_camera</depend>
+  <depend>realsense2_camera_msgs</depend>
```

Comment:
General thought on specific hardware: sooner or later we will end up with a lot of dependencies here. Is it possible to have optional dependencies? I am thinking that you can run the default setup (no lidar nor camera) without them. The specific hw nodes only need to be available when you acutally want to enable them.

Reply:
It is a good point. I remove these hardware related dependencies from package.xml and added them in readme file.